File: sdkwork-backend-sdk-go/types/plus_vip_pack_group_vo.go
package types

// VIP套餐分组VO
type PlusVipPackGroupVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id int `json:"id"`
	Uuid string `json:"uuid"`
	AppId int `json:"appId"`
	Name string `json:"name"`
	Description string `json:"description"`
	SortWeight int `json:"sortWeight"`
	Status string `json:"status"`
	Remark string `json:"remark"`
	Packs []PlusVipPackVO `json:"packs"`
}

File: sdkwork-backend-sdk-csharp/Models/PlusVipPackGroupVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class PlusVipPackGroupVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public int? Id { get; set; }
        public string? Uuid { get; set; }
        public int? AppId { get; set; }
        public string? Name { get; set; }
        public string? Description { get; set; }
        public int? SortWeight { get; set; }
        public string? Status { get; set; }
        public string? Remark { get; set; }
        public List<PlusVipPackVO>? Packs { get; set; }
    }
}
